neural_network/activation_functions/sobovela_modified_hyperbolic_tangent.py

```diff
@@ -0,0 +1,47 @@
+"""
+This script implements the Sobovela Modified Hyperbolic Tangent function.
+
+The function applies the Sobovela Modified Hyperbolic Tangent function
+to each element of the vector.
+
+More details about the activation function can be found on:
+https://en.wikipedia.org/wiki/Soboleva_modified_hyperbolic_tangent
+"""
+
+
+import numpy as np
+
+
+def sobovela_modified_hyperbolic_tangent(
+    vector: np.ndarray, a_value: float, b_value: float, c_value: float, d_value: float
+) -> np.ndarray:
+    """
+    Implements the Sobovela Modified Hyperbolic Tangent function
+
+    Parameters:
+        vector (ndarray): A vector that consitis of numeric values
+        alpha (float): alpha value
+        beta (float): beta value
```

Comment:
This section doesn't match the actual parameters of the function

Reply:
Thanks